### ISSUE TITLE
Clarify dependency "story" and installation procedure

### ISSUE THREAD

**Issue by cisaacstern** [leap-stc/ClimSim#25]
At a glance, it's quite confusing currently how/what one is supposed to install in the environment to make the code in this repo work. Let's clarify that "story". Some items to fix:

- [ ] It seems reasonable for a user to expect that installing `climsim_utils` (via `pip install .`) should give them everything they need. Currently, each of those directories has a `requirements-lock.txt` which @jerrylin96 has indicated is for development purposes (as a reference of a known working environment). Once development on those notebooks is complete (i.e., #21 is closed), let's try to get those notebooks to run based on only dependencies in `setup.py`.
- [ ] The `environment.yml` in the repo root is actually just for the website. That's super confusing 😄 .
- [ ] Each of the `baseline_models` has it's own `env/`. That's presumably desirable, as we would not expect the models to run with the same env as the preprocessing/evaluation code. Perhaps we can just noted in some way in the README (if that is not already done).

**Comment by NoraLoose** [comment]
> The `environment.yml` in the repo root is actually just for the website. That's super confusing 😄 .

Exactly. Therefore I suggest to move `environment.yml` to the `website` directory.

**Comment by cisaacstern** [comment]
> Exactly. Therefore I suggest to move environment.yml to the website directory.

💯 agree

**Comment by cisaacstern** [comment]
https://github.com/leap-stc/ClimSim/pull/30 relocates `environment.yml` as discussed above, I will check it off the to-do list once that's merged.
